plugins/plugin-chart-choropleth-map/src/chart/ChoroplethMap.tsx

```diff
@@ -92,11 +125,91 @@ export default class ChoroplethMap extends React.PureComponent<
 
   render() {
     const { height, width } = this.props;
+    const { showMiniMap } = this.state;
+
+    const renderedMap = this.renderMap();
+    const miniMapTransform = `translate(${(width * 3) / 4 - PADDING}, ${
+      (height * 3) / 4 - PADDING
+    }) scale(0.25)`;
 
     return (
-      <svg width={width} height={height}>
-        {this.renderMap()}
-      </svg>
+      <Zoom
+        style={{ width, height }}
+        width={width}
+        height={height}
```

Comment:
Is it required to have both width/height prop and style width/height for vx/zoom to work?

Reply:
Yes, probably can file an issue to `@vx/zoom` too. The `width` and `height` attributes are used to compute the zoom transform, but it does not set the container (`<div>`) size correctly, had to specify style for that.